kerl

```diff
@@ -311,109 +310,108 @@ get_releases() {
 
 get_git_releases() {
     tmp="$(mktemp "$TMP_DIR"/kerl.XXXXXX)"
-    git ls-remote --tags --refs "$OTP_GITHUB_URL" > "$tmp"
+    git ls-remote --tags --refs "$OTP_GITHUB_URL" >"$tmp"
     ret=$?
-    if [ $ret -eq 0 ]; then
-        < "$tmp" cut -f2 \
```

Comment:
Still checking why the formatter would consider `< "$tmp" cut -f2` equivalent to `cut <"$tmp" -f2`...

Reply:
Yup, they're equivalent.